lib/smart_proxy_ansible/roles_reader.rb

```diff
@@ -7,20 +7,32 @@ class RolesReader
       class << self
         DEFAULT_CONFIG_FILE = '/etc/ansible/ansible.cfg'.freeze
         DEFAULT_ROLES_PATH = '/etc/ansible/roles:/usr/share/ansible/roles'.freeze
+        DEFAULT_COLLECTIONS_PATHS = '/etc/ansible/collections'.freeze
```

Comment:
The [default paths](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths) for collections are ~/.ansible/collections:/usr/share/ansible/collections, I would add at least the second one here as well.